src/URIHandler.js

```diff
@@ -0,0 +1,36 @@
+import { handler } from './handlerUtil';
+
+const NAMESPACE = /^[^]*[#/]/;
+const FRAGMENT = /(?![/#])[^/#]*$/;
```

Comment:
This one is not correct. (`http://abc/def#abc/def`)
Perhaps best to use `NAMESPACE` to strip off the beginning; or change it to have 2 capturing groups.